pkg/database/cosmosdb/zz_generated_authorizer.go

```diff
@@ -15,14 +16,14 @@ import (
 )
 
 type Authorizer interface {
-	Authorize(*http.Request, string, string) error
+	Authorize(context.Context , *http.Request, string, string) error
```

Comment:
Nit: `Authorize(context.Context, *http.Request, string, string) error`

Extra space between `Context` and `.`

Reply:
Done.